src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
+    UserDao userDao = new UserDaoImpl();
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            int value = 0;
+            while (true) {
+                System.out.println("Enter value:");
+                try {
+                    value = Integer.parseInt(scanner.nextLine());
+                    break;
```

Comment:
program will stop if user will enter a valid value? how about risk then?

Reply:
no? program won't stop, operator break is inside while (true) loop of value, risk has it own loop